lib/openshift/route.rb

```diff
@@ -106,5 +106,13 @@ def dns(by: nil, user: nil, cached: true, quiet: false)
       return raw_resource(user: user, cached: cached, quiet: quiet).
         dig("spec", "host")
     end
+    # check for ingress route only
+    def ready?(user: nil, quiet: false)
+      status = raw_resource(user: user, cached: false, quiet: quiet)['status']
+      ready = status.dig('ingress').last['conditions'].any? do |con|
```

Comment:
dig only needed for multiple elements, otherwise `['something']` works the same an is nicer. You may want to use `.&` instead of `.` though to avoid exceptions when `ingress` element is missing.